src/ResourceManager/Compute/Commands.Compute/Extension/VMAccess/VirtualMachineAccessExtensionContext.cs

```diff
@@ -23,6 +23,9 @@ public class VirtualMachineAccessExtensionContext : PSVirtualMachineExtension
         public const string ExtensionDefaultPublisher = "Microsoft.Compute";
         public const string ExtensionDefaultName = "VMAccessAgent";
         public const string ExtensionDefaultVersion = "2.0";
+        public const string ExtensionLinuxPublisher = "Microsoft.OSTCExtensions";
+        public const string ExtensionLinuxName = "VMAccessForLinux";
+        public const string ExtensionLinuxVersion = "2.0";
```

Comment:
Why is this 2.0?  The latest version of VMAccessForLinux is 1.4.6.0.

How was this tested?